openshift/openshift-ansible-contrib
# Sync only selected images for Sat6.

#### What does this PR do?
Instead of syncing things that are not used and missing that that are needes.
Lets sync from the list provided by a redhat KB entry.

#### How should this be manually tested?
Setup a Sat6 and install OCP 3.11 against this.
See: https://access.redhat.com/solutions/3435901 and https://blog.openshift.com/using-satellite-6-server-disconnect-openshift-container-platform-install/

#### Is there a relevant Issue open for this?
No

#### Who would you like to review this?
cc: @tomassedovic PTAL
